tests/unit/CorrectNameRequest.spec.ts

```diff
@@ -15,7 +15,41 @@ function getLastEvent (wrapper: Wrapper<any>, name: string): any {
   return null
 }
 
-describe.skip('CorrectNameRequest', () => {
+const nameRequest = {
+  state: 'APPROVED',
+  expirationDate: '2022-05-19',
+  names: [{
+    state: 'APPROVED',
+    name: 'Bobs Plumbing'
+  }],
+  nrNum: 'NR 1234567',
+  requestTypeCd: 'BC',
+  legalType: 'BC',
+  request_action_cd: 'CNV',
+  applicants: {
+    phoneNumber: '250 516 8257',
+    emailAddress: 'mock@example.com'
+  }
+}
+
+const fetchAndValidateNr = (nrNumber, businessId, applicantPhone, applicantEmail) => {
+  if(nrNumber != nameRequest.nrNum) return null
+  if(applicantEmail && applicantEmail != nameRequest.applicants.emailAddress) return null
+  if(applicantPhone && applicantPhone != nameRequest.applicants.phoneNumber) return null
+
+  return Promise.resolve(nameRequest)
+}
```

Comment:
I don't know well how to mock this function.
So, I made the `fetchAndValidateNr` like that based on the test case.

Reply:
ok